src/implementation/Client/HTTPClient/HTTPClient.ts

```diff
@@ -11,16 +11,16 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-import fetch from "node-fetch";
-import { CommunicationProtocolEnum, DaprClient } from "../../..";
-import IClient from "../../../interfaces/Client/IClient";
-import http from "http";
-import https from "https";
-import { DaprClientOptions } from "../../../types/DaprClientOptions";
+import fetch from 'node-fetch';
+import { CommunicationProtocolEnum, DaprClient } from '../../..';
+import IClient from '../../../interfaces/Client/IClient';
+import http from 'http';
+import https from 'https';
+import { DaprClientOptions } from '../../../types/DaprClientOptions';
 import { Settings } from '../../../utils/Settings.util';
-import { THTTPExecuteParams } from "../../../types/http/THTTPExecuteParams.type"
-import { Logger } from "../../../logger/Logger";
-import HTTPClientSidecar from "./sidecar";
+import { THTTPExecuteParams } from '../../../types/http/THTTPExecuteParams.type';
+import { Logger } from '../../../logger/Logger';
+import HTTPClientSidecar from './sidecar';
```

Comment:
Lot of similar changes in this PR, can you please revert them @ahmedhesham6? I am okay with these but we should track it separately + configure ESLint. 

Also looks like there are some extra commits in this PR.

Reply:
I don't know what is the issue with linting so I will reopen this pr

Reply:
Done please review.